project/AppDependencies.scala

```diff
@@ -4,12 +4,17 @@ import sbt._
 object AppDependencies {
 
   val compile = Seq(
-    "uk.gov.hmrc" %% "simple-reactivemongo"          % "7.22.0-play-26",
-    "uk.gov.hmrc" %% "govuk-template"                % "5.48.0-play-26",
-    "uk.gov.hmrc" %% "play-json-union-formatter"     % "1.7.0",
-    "uk.gov.hmrc" %% "play-ui"                       % "8.5.0-play-26",
-    "uk.gov.hmrc" %% "bootstrap-play-26"             % "1.3.0",
-    "uk.gov.hmrc" %% "play-conditional-form-mapping" % "1.2.0-play-26",
+    "uk.gov.hmrc"          %% "play-conditional-form-mapping" % "1.2.0-play-26",
+    "uk.gov.hmrc"          %% "logback-json-logger"           % "4.6.0",
+    "uk.gov.hmrc"          %% "govuk-template"                % "5.48.0-play-26",
+    "uk.gov.hmrc"          %% "play-health"                   % "3.14.0-play-26",
+    "uk.gov.hmrc"          %% "play-ui"                       % "8.5.0-play-26",
+    "uk.gov.hmrc"          %% "bootstrap-play-26"             % "1.3.0",
+    "uk.gov.hmrc"          %% "play-frontend-govuk"           % "0.32.0-play-26",
+    "org.webjars.npm"      %  "govuk-frontend"                % "3.4.0",
+    "uk.gov.hmrc"          %% "play-json-union-formatter"     % "1.7.0",
+    "uk.gov.hmrc"          %% "simple-reactivemongo"          % "7.22.0-play-26",
+    "ai.x"                 %% "play-json-extensions"          % "0.40.2",
```

Comment:
What are `play-health` and `play-json-extensions` used for? are they really needed?

Reply:
I'll test the need for this in the next page affected by this ticket, to be in a PR today.